src/chart/bar/BarChart.tsx

```diff
@@ -71,46 +96,15 @@ const NeoBarChart = (props: ChartProps) => {
         });
         return row;
       });
-
     setKeys(Object.keys(newKeys));
     setData(newData);
-  }, [selection]);
 
-  if (loading) {
-    return <></>;
-  }
+  }, [selection, minBarHeight]);
```

Comment:
Why is minBarHeight explicitly called out here?
In the `PieChart` we have this useEffect being triggered by `[selection, records]`, perhaps we can do the same here.

Reply:
Removed minBarHeight